netmap/netmap_contract.go

```diff
@@ -601,6 +605,32 @@ func ListConfig() []record {
 	return config
 }
 
+// SubscribeForNewEpoch registers passed contract as a NewEpoch event
+// subscriber. Such a contract must have a `NewEpoch` method with a single
+// numeric argument. Transactions that call SubscribeForNewEpoch must be
+// witnessed by the Alphabet.
+// Produces `NewEpochHookRegistration` notification event with a just
```

Comment:
according to code it is
```suggestion
// Produces `NewEpochSubscription` notification event with a just
```

Reply:
fixed. it did not live renamings well